tests/pyfunc/test_spark.py

```diff
@@ -1045,3 +1046,14 @@ def predict(self, context, model_input):
         data1 = spark.range(3).repartition(1)
         result = data1.select(udf("id").alias("res")).select("res.a").toPandas()
         assert list(result["a"]) == [[1.0, None], None, None]
+
+
+def test_spark_udf_with_spark_connect(sklearn_model, model_path):
+    model, data = sklearn_model
+    mlflow.sklearn.save_model(model, model_path)
+    spark = SparkSession.builder.remote("local[2]").getOrCreate()
+    data = pd.DataFrame(data, columns=["a", "b"]).head(3)
+    sdf = spark.createDataFrame()
+    udf = spark_udf(spark, model_path, env_manager="local")
+    result = sdf.select(udf("a", "b").alias("preds")).toPandas()
```

Comment:
But the test does not cover the path that downloading model in worker side (because client and worker shares the same disk)
Can we add a config for testing purpose to address the issue ?

Reply:
> Can we add a config for testing purpose to address the issue ?

Can you elaborate on this?

Reply:
i.e. The config (or environment variable setting) can force downloading model in worker side, even if client and worker shares the same disk